cli/ballerina-tool/src/main/resources/cli-help/ballerina-test.help

```diff
@@ -1,105 +1,51 @@
 NAME
-       Ballerina test - Run Ballerina tests and display a summary of
-       test results.
+       ballerina test - execute ballerina tests
 
 SYNOPSIS
-       ballerina test [-lg|--list-groups] [--groups <test_group, ...>]
-           [--disable-groups <test_group, ...>] [--sourceroot <path>]
-           [-e] [--debug <port>] [-c <config_file>]
-           [--exclude-modules <modules, ...>]
-           [--config <config_file>] [-B] [<balfile>|<module-name>]
+       ballerina test [--offline]
+           [--sourceroot <path>]
+           [--experimental][--skip-lock]
+           [--config <config_file>]
+           [<module-name>|-a|--all]
 
 
 DESCRIPTION
        Ballerina test command allows you to compile and run Ballerina
        test sources and prints a summary of test results. You can
-       execute tests in a single test file, a test module, or an
-       entire Ballerina project.
-    
-       Use ballerina test without any parameters to execute tests for
-       the entire project.
+       execute tests in a test module, or an entire Ballerina project.
     
        Use ballerina test <module-name> to execute tests within the
-       specified module.
-    
-       Use ballerina test <balfile> to execute the given Ballerina test
-       file. Note that imports from other modules will not work for
-       file executions. File path can be a relative or absolute.
+       specified module, or -a or -all to execute tests in all modules.
 
 
 OPTIONS
+        --offline
+            In a module build, the dependencies are always checked from
```

Comment:
Shall we make this uniform with https://github.com/ballerina-platform/ballerina-lang/pull/18834/files#diff-fd96dc1ef9c62010f9406cc6373ea3f4R36

Reply:
I took it form the commands help text, but let's go with your suggestion.